eclair-core/src/test/scala/fr/acinq/eclair/TestConstants.scala

```diff
@@ -43,6 +46,23 @@ object TestConstants {
   val pushMsat = 200000000L msat
   val feeratePerKw = 10000L
   val emptyOnionPacket = wire.OnionRoutingPacket(0, ByteVector.fill(33)(0), ByteVector.fill(1300)(0), ByteVector32.Zeroes)
+  val postgreConfig = ConfigFactory.parseString("""eclair = {
+                                                  |    postgre = {
+                                                  |      connectionPool = "HikariCP"
+                                                  |      dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
+                                                  |      numThreads = 6
+                                                  |
+                                                  |      properties = {
+                                                  |        serverName = "localhost"
+                                                  |        portNumber = "5432"
+                                                  |        databaseName = "eclairtest"
+                                                  |        user = "postgres"
```

Comment:
This file need editing for running tests. Any Idea how to make the tests working without changing the config and installing `postgresql`?